extensions/stackdriver/stackdriver.cc

```diff
@@ -107,6 +107,16 @@ int getExportInterval() {
   return 60;
 }
 
+// Get port of security token exchange server from node metadata, if not
+// provided, emtpy will be returned.
+std::string getSTSPort() {
+  std::string sts_port = "";
+  if (getValue({"node", "metadata", kSTSPortKey}, &sts_port)) {
```

Comment:
My PR which adds STS flag is here. We Pilot-agent needs to check this flag and add metadata into bootstrap config.

Reply:
ptal at istio/istio#20595